### PR TITLE
add control detector logic

### PR DESCRIPTION
**Pull request by phuongdv88**
create and delete detector when start/stop the client handling thread

### CODE REVIEW

**Comment by phuongdv88** [comment]
Please don't merge it until I test it tomorrow afternoon

**Comment by phuongdv88** [comment]
Dear @zooyeon  It's ready to merge

File: LgClientDisplay_python/dist/LgClientController.exe
Comment (zooyeon) [inline LgClientDisplay_python/dist/LgClientController.exe:1]: @phuongdv88 
do you want to delete LgClientContoroller.exe?

@rdg5924

**Review by zooyeon — CHANGES_REQUESTED** [review]
@phuongdv88 why do you want to delete LgClientController.exe?

File: LgClientDisplay_python/dist/LgClientController.exe
Comment (phuongdv88) [reply LgClientDisplay_python/dist/LgClientController.exe:1]: @zooyeon my antivirus app deleted it. Not me

Comment (zooyeon) [reply LgClientDisplay_python/dist/LgClientController.exe:1]: @phuongdv88 Okay, but please do not include it in the commits.

**Pull request merged**